src/VisualStudio.Roslyn.SDK/Roslyn.SDK/BindingPathProvider.cs

```diff
@@ -0,0 +1,17 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.VisualStudio.Shell;
+
+namespace Roslyn.SDK
+{
+    /// <summary>
+    /// Ensures that Visual Studio probes the package directory for assemblies at load time.
+    /// </summary>
+    /// <seealso cref="https://docs.microsoft.com/en-us/dotnet/api/microsoft.visualstudio.shell.providebindingpathattribute"/>
+    [ProvideBindingPath]
+    internal class BindingPathProvider
```

Comment:
We need a way to tell VS to look in the package directory to find assemblies for the component debugger rule. We don't have a package for the debugger itself, we just embed it in the SDK package, so this adds the required registry entries for it. Annoyingly you can't put it at the assembly level so we have to have a dummy class to put it on.

Reply:
❗ Use `ProvideCodeBase` instead of `ProvideBindingPath`

Reply:
Ok, I have no idea, was just following the instructions in the doc linked in the description. What's the difference / what does it do?

Reply:
> What's the difference / what does it do?

ProvideBindingPath means every assembly load will look in this extension folder to see if it contains the assembly. ProvideCodeBase restricts the new search location to only apply to one specific assembly (better performance).

Reply:
Gotcha, thanks for the explanation!